demo-ui/test/uk/gov/ons/addressIndex/demoui/client/AddressIndexClientMock.scala

```diff
@@ -26,11 +26,12 @@ class AddressIndexClientMock @Inject()(override val client : WSClient,
     message = "OK"
   )
 
-  val mockAddressTokens = AddressTokens(
-    uprn = "",
-    buildingNumber = "7",
-    postcode = "EX2 9GA"
-  )
+  val mockAddressTokens = Seq.empty
+//  AddressTokens(
```

Comment:
I didnt want to loose anything in the file, especially as this is the test input, this test is now ignored, so when we come round to fixing it, we can see the input